civis/tests/test_polling.py

```diff
@@ -71,8 +71,8 @@ def test_poll_on_creation(self):
                                   poll_on_creation=False)
         pollable.done()  # Check status once to start the polling thread
         assert poller.call_count == 0
-        time.sleep(0.015)
-        assert poller.call_count == 1
+        time.sleep(0.02)
+        assert poller.call_count > 0
```

Comment:
Because now we have different execution environments for builds (Circle CI versus AppVeyor), this test has become a little unstable (see this [test failure](https://ci.appveyor.com/project/eabraun/civis-python/build/1.0.9/job/9nn5y16e1ctvjqrk) due to such indeterminancy). Given `polling_internal=0.01` (set just a few lines above, via `PollableResult`),  sleeping 0.02s gives a little more time for at least one poller call count (if not two).

Reply:
It was a little unstable in the Linux tests, to be honest. Hopefully this change makes life better for everyone. Thank you!